bioimageio/spec/v0_3/raw_nodes.py

```diff
@@ -2,11 +2,12 @@
 from dataclasses import dataclass
 from datetime import datetime
 from pathlib import Path
-from typing import Any, Dict, List, NewType, Tuple, Union
+from typing import Any, ClassVar, Dict, List, NewType, Tuple, Union
 
 from marshmallow import missing
 from marshmallow.utils import _Missing
 
+from bioimageio.spec import v0_1
```

Comment:
should this then be maybe moved to common? I wouldn't expect v0_1 imports here.

Reply:
why not? v0_3 can reuse definitons from v0_1, no?

As there is a possiblity to convert from 0.1.x to 0.3.x format version is defined below as:
```
ModelFormatVersion = Literal[
    v0_1.ModelFormatVersion, "0.3.0", "0.3.1", "0.3.2"  # newest format needs to be last (used in spec.__init__.py)
]
```

I changed this as the current 'root' spec is the latest, which should include previous versions to the extent that is supported. E.g. we might break this chain with major release bumps.